portfolio/src/main/webapp/script.js

```diff
@@ -42,7 +42,10 @@ function getMessage() {
   fetch('/data').then(response => response.json()).then((messages) => {
     console.log(messages);
     let html = '';
-    let numDisplay = getParameter("numComments", 10);
+    let numDisplay = getParameter("numComments");
+    if (numDisplay == null) {
+      numDisplay = 10; // Default value
```

Comment:
[Nit] Inline comment should have 2 spaces after the semicolon and end with a '.'